app/src/main/java/net/gsantner/opoc/util/StringUtils.java

```diff
@@ -690,4 +692,20 @@ public CharSequence subSequence(int start, int end) {
             return select().subSequence(start, end);
         }
     }
+
+    public static Runnable makeDebounced(final long delayMs, final Runnable callback) {
```

Comment:
This should probably not be in `StringUtils`. But I can't think of a better place to put it.